### PR TITLE
Minor tweaks to improve flow and clarity

### PR DESCRIPTION
**Pull request by gregorgorjanc**
I read the `what_is.md` the other day and thought these edits would improve flow and add clarity for those that lack some training in tree-stuff. Take what you find useful.

### CODE REVIEW

File: what_is.md [hunk 1]
```diff
@@ -74,9 +74,9 @@ def create_notebook_data():
 
 A *succinct tree sequence*, or "tree sequence" for short, represents the ancestral
 relationships between a set of DNA sequences. Tree sequences are based on fundamental
-biological principles of inheritance, DNA duplication, and recombination; they can be
-created by [evolutionary simulation](https://tskit.dev/software/#simulate) or by
-[inferring genealogies from empirical DNA data](https://tskit.dev/software/#infer).
+biological principles of inheritance, DNA duplication, mutation, and recombination;
```
Comment (gregorgorjanc) [inline what_is.md:77]: Just to make it clear that mutation is also important.

File: what_is.md [hunk 2]
```diff
@@ -85,8 +85,9 @@ Tree sequences are used to encode and analyse large genetic datasets
 Tree sequences provide an efficient way of storing
 [genetic variation](https://en.wikipedia.org/wiki/Genetic_variation) data, and can
 power analyses of millions of whole [genomes](https://en.wikipedia.org/wiki/Genome).
-Plots (a) and (b) summarize results presented
-[further](plot_storing_everyone) [down](plot_incremental_calculation) this tutorial.
+Plots (a) and (b) below summarize these aspects
```
Comment (gregorgorjanc) [inline what_is.md:88]: I found this flow a bit awkward.

**Comment by benjeffery** [comment]
Thanks, all LGTM. I'll wait for @hyanwong to merge as I think he wrote the original

File: what_is.md [hunk 3]
```diff
@@ -355,10 +363,10 @@ tree sequence and the underlying biological processes that produced the genetic
 sequences in the first place, such as those pictured in the demography above. For
 example, each branch point (or "internal node") in one of our trees can be
 imagined as a genome which existed at a specific time in the past, and
-which is a "most recent common ancestor" (MRCA) of the descendant genomes at that
-position on the chromosome. We can mark these extra "ancestral genomes" on our tree
-diagrams, distinguishing them from the *sampled* genomes ($\mathrm{a}$ to $\mathrm{j}$)
-by using circular symbols. We can even colour the nodes by the population that we know
+which is a MRCA of the descendant genomes at that position on the chromosome.
```
Comment (hyanwong) [inline what_is.md:366]: We don't define MRCA anywhere previously, do we? So I think we do need to spell it out here.

File: what_is.md [hunk 4]
```diff
@@ -187,8 +195,8 @@ In fact, succinct tree sequences don't store each tree separately, but instead a
 based on an interconnected *genetic genealogy*, in which
 [genetic recombination](https://en.wikipedia.org/wiki/Genetic_recombination) has led
 to different regions of the chromosome having different histories. Another way of
-thinking about the tree sequence above is that it describes the full genetic
-*family "tree"* (strictly, "network") of our 10 genomes.
```
Comment (hyanwong) [inline what_is.md:191]: I think it's helpful to mention "network" somewhere, as that's in the title. I also think a "genetic family tree" is worth mentioning somewhere, but perhaps not here. I'm happy to merge this and add something like this sentence in again at the top.

**Review by hyanwong — APPROVED** [review]
LGTM generally. Happy to merge and add in any extra changes later

**Comment by hyanwong** [comment]
> Thanks, all LGTM. I'll wait for @hyanwong to merge as I think he wrote the original

I don't think I have permissions to merge on this repo @benjeffery. Perhaps I should have?

**Comment by benjeffery** [comment]
> I don't think I have permissions to merge on this repo @benjeffery. Perhaps I should have?

Invite sent!

**Pull request merged**